spec/ruby/i18n/js_spec.rb

```diff
@@ -307,21 +307,17 @@
       expect(subject[:de][:null_test]).to eql(nil)
     end
 
-    context "when given locale is in `I18n.available_locales` but its translation is missing" do
```

Comment:
I think the original test cases should be kept.  
And copy `I18n.available_locales` test cases to a new context for `js_available_locales` and modify those copied test cases for `js_available_locales`.